template.rb

```diff
@@ -381,19 +381,21 @@ def cleanup_package_json
   package_json.manager.install!
 end
 
-# Adds the given <code>packages</code> as dependencies using <code>yarn add</code>
+# Adds the given JavaScript <code>packages</code> as dependencies using the
+# appropriate package manager
 #
 # @param [Array<String>] packages
-def yarn_add_dependencies(packages)
+def add_js_dependencies(packages)
   puts "adding #{packages.join(" ")} as dependencies"
 
   package_json.manager.add!(packages)
 end
 
-# Adds the given <code>packages</code> as devDependencies using <code>yarn add --dev</code>
+# Adds the given JavaScript <code>packages</code> as devDependencies using the
+# appropriate package manager
 #
 # @param [Array<String>] packages
-def yarn_add_dev_dependencies(packages)
+def add_js_dev_dependencies(packages)
```

Comment:
Have you considered merging these methods? `add_js_dev_dependencies(packages, dev: true)`

Reply:
I think I didn't do that in the past for the look and aso because I was planning to replace it with `package_json.manager.add!` but then realized it would be nice to keep the logging...

I've just gone and made that change since I don't have a good reason not to 🤷